### DISCUSSION TITLE
Chapter 5 Notes and Discussion

### DISCUSSION THREAD

**Discussion by voulacol** [Zibiana/QR4SJ#5] [General]
In Section 5.1, the genderbread person link isn't functional, so should be replaced. Bianca found a new link

**Comment by Zibiana** [comment]
New link, I'll update the link in the book later: https://www.itspronouncedmetrosexual.com/genderbread-person/ 

As another thing I'd like to fix: Can we make ch 5.1 "exercise group" into a "Preactivity" or something? That feels like something students may wish to think about and work on before walking in the door.

**Comment by voulacol** [comment]
The tables in section 5.3.1 are not in the most understandable format. I think it would be helpful to rework these so they are more easily understood.

**Comment by voulacol** [comment]
Section 5.3.1.1 proposes a question, but the following explanation feels out of order and incomplete. I think the structure of this section and the framing could be improved and built out more.

**Comment by voulacol** [comment]
Section 5.3.2. The language in this section will need to be updated once it is no longer 2023. Perhaps this would also be a good place for introduce an exercise like "Based on the data collected through April 21, 2023, estimate the number of of anti-trans bills that were introduced over the entire year." And then (once 2023 has ended) you could tell the students the actual numbers and then get them to come up with reasons why their estimate might not match the actual numbers.

**Comment by voulacol** [comment]
I'm not sure I completely understand the purpose of section 5.4. Is this something to introduce students to different ideas that they could investigate on their own, and if so, should there be an explanation of what we mean by Bayesian analysis? Is this a place to compile ideas that we will flesh out later to become new sections to the book?

**Reply by voulacol** [reply]
This was discussed and this section will be moved to the back matter

**Comment by jehibdon** [comment]
Section 1: Comments and Suggestions - 
The references that are being used should also have a hyper link to the References section or link the paper/websites themselves.  Makes it easier for students to pull up the primary resource.

**Comment by jehibdon** [comment]
Section 2: Comments and Suggestions - 
"(that you won’t have to worry about!)" In subsection 2 should be a possibility for students to view this code if just out of curiosity.  I find this wording condescending even if it is true.  Maybe mention that it has been done and then give a link to the github with the code that does the transformation.

**Comment by RachelRoca** [comment]
Section 1 comment: Wondering if we should make the categories of bills paragraphs into a table? Might be more readable?
